docs/_documentations/che-openshift-certs.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: docs
+title: Downloading Openshift certificates
+description: 
+keywords: build, deploy, install, installing, installation, chart, Helm, develop, cloud, public cloud, services, command line, cli, command, start, stop, update, open, delete, options, operation, devops, OpenShift, OKD, root, privileged
+duration: 1 minute
+permalink: che-openshift-certs
+type: document
+---
+# Downloading Openshift certificates
+
+Ensure you have completed the [prerequisites to installing Codewind on Eclipse Che](./che-installinfo.html#prerequisites).
+
+Follow these instructions to download the Openshift router certificate: 
+
+1. Authenticate against your OpenShift cluster, or ask your cluster administrator to do so.
```

Comment:
These steps need to be performed by a user who has logged into the Openshift cluster,  either the user should do it or if they can't then the administrator should do it on their behalf.  The end result is that once logged in they run the commands gather the certificates and then provide them back to the user for them to install into their browsers. In most cases they could just publish these certificates on a download site.  There is also an alternative to these steps which do not require logging into the cluster at all.  That would be to use  openssl and extract the public certificate.  The TLS handshake steps complete prior to authentication so you can grab all the information you need without actually needing to log in, but for now this is fine.

Reply:
File is no longer in the ToC nor referenced from other ToCs. Forgot to delete file - raising a PR to delete this.